streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredWindowStoreTest.java

```diff
@@ -272,21 +230,8 @@ public void shouldRecordRestoreLatencyOnInit() {
         expectLastCall();
         replay(innerStoreMock);
         store.init((StateStoreContext) context, store);
-        final Map<MetricName, ? extends Metric> metrics = context.metrics().metrics();
-        if (StreamsConfig.METRICS_0100_TO_24.equals(builtInMetricsVersion)) {
-            assertEquals(1.0, getMetricByNameFilterByTags(
-                metrics,
-                "restore-total",
-                storeLevelGroup,
-                singletonMap(STORE_TYPE + "-state-id", STORE_NAME)
-            ).metricValue());
-            assertEquals(1.0, getMetricByNameFilterByTags(
-                metrics,
-                "restore-total",
-                storeLevelGroup,
-                singletonMap(STORE_TYPE + "-state-id", ROLLUP_VALUE)
-            ).metricValue());
-        }
+
+        verify(innerStoreMock);
```

Comment:
We forgot to add this before?

Reply:
Good that you mention that! I checked the metered state store tests and found some testing holes.